src/overlays/actors/ovl_Obj_Chan/z_obj_chan.c

```diff
@@ -31,49 +39,401 @@ const ActorInit Obj_Chan_InitVars = {
     (ActorFunc)ObjChan_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BBAB30 = {
-    { COLTYPE_HARD, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_NONE, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK1, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+static ColliderCylinderInit sObjChanCylinderInit = {
+    {
+        COLTYPE_HARD,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK1,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_NONE,
+    },
     { 48, 76, -60, { 0, 0, 0 } },
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80BBAB5C[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+void ObjChan_InitMain(ObjChan* this, GlobalContext* globalCtx); /* static */
+void ObjChan_InitPot(ObjChan* this, GlobalContext* globalCtx);  /* static */
+void func_80BBA488(ObjChan* this, GlobalContext* globalCtx);    /* static */
+void ObjChan_DrawPot(Actor* thisx, GlobalContext* globalCtx);   /* static */
+void func_80BBA930(ObjChan* this, GlobalContext* globalCtx);    /* static */
+
+static Vec3f sObjChanFlameSize[2] = { { 0.159999996424f, 0.109999999404f, 1.0f },
+                                      { 0.129999995232f, 0.0900000035763f, 1.0f } };
```

Comment:
```suggestion
static Vec3f sObjChanFlameSize[2] = { { 0.159999996424f, 0.109999999404f, 1.0f },
                                      { 0.129999995232f, 0.0900000035763f, 1.0f }, };
```
This will probably format better with a trailing comma.

Reply:
Indeed it does.